### PR TITLE
TensorFlow 2D encoding added

### PR DESCRIPTION
**Pull request by IRiViI**
Hey, thanks for the code. I have most of my code in TensorFlow/Keras already so I converted your 2D encoding from torch to TensorFlow. I think other might also find this useful. 

Cheers

### CODE REVIEW

**Comment by tatp22** [comment]
Hey @IRiViI! Thanks for the contribution!

I agree, i think that others might find this useful as well. Ill make a quick change to this pr as well (mainly cleanups and more changes to the readme), and then I'll merge it.

**Comment by IRiViI** [comment]
Cool, let me know when you are ready. I already found a bug in my code (which I just fixed).

File: positional_encodings/tf_positional_encodings.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+import tensorflow as tf
+import numpy as np
+
+import tensorflow as tf
```
Comment (tatp22) [inline positional_encodings/tf_positional_encodings.py]:
```suggestion
import tensorflow as tf
import numpy as np
```

**Review by tatp22 — CHANGES_REQUESTED** [review]
You know what, I can just redo the readme in another PR.

I just had one quick suggestion, theres a redundant import i think, but other than that it looks good :+1:. I'll merge it then

**Comment by IRiViI** [comment]
Oke cool, I only added a small part to the readme.

Well spotted the import redundancy. I guess that is what you get when copy pasting your code.

File: positional_encodings/tf_positional_encodings.py (hunk 1 above)
Comment (IRiViI) [reply positional_encodings/tf_positional_encodings.py]: I resolved it in the editor.

**Comment by tatp22** [comment]
Cool, thank you! I'll merge this.

**Comment by IRiViI** [comment]
Awesome! 

I could probably also do the same for 1D and 3D but I didn't need it though. 

There is one small difference in the outcome between the TensorFlow en PyTorch version. the outcome has a difference in the order of e-8 when I tested it. So I think it it due to rounding when the data was in 8 bit format or something. I don't care enough to track down where it happens but let me know if you think this should also be addressed. 

Cheers,

**Comment by tatp22** [comment]
Nah, I think thats fine. Actually, I can try myself to see if i can do the 1 and 3d version as a learning experience.

Also, i think that this small rounding error is fine, as it is not so much.

**Pull request merged**